docs/src/css/custom.css

```diff
@@ -22,8 +22,8 @@
   --ifm-color-primary-lighter: #359962;
   --ifm-color-primary-lightest: #3cad6e;
   --ifm-code-font-size: 85%;
-  --ifm-container-width: 800px;
-  --ifm-container-width-xl: 800px;
+  /* --ifm-container-width: 800px;
```

Comment:
Could we remove entirely or add a comment about why this is commented out

Reply:
removed